### ISSUE TITLE
New fset argument doesn't default to NULL

### ISSUE THREAD

**Issue by ben-wetherill** [massbays-tech/MassWateR#36]
I see that you added the fset argument.  It works well everywhere, but the qcMWRreview() function results in an error if fset is not defined, even though res, acc, and frecom are defined.  fset doesn't default to NULL.
`qcMWRreview(resdat,dqoaccdat,dqofrecomdat, "D:/Documents/", output_file = "QCReport", rawdata = FALSE)`

Also, slightly related, in all of the functions if you don't define any source tables (i.e. you forget to define res, acc, frecom, sit, and fset), you get the following criptic error.  Is it possible to make the error for this situation more understandable, such as "Data tables were not defined".
`Error` in UseMethod("filter") : no applicable method for 'filter' applied to an object of class "NULL"`

**Comment by fawda123** [comment]
Hey @ben-wetherill, the issue with the example code above is you did not make the argument for the output location explicit.  R is thinking that you're trying to pass `"D:/Documents/"` to the `fset` argument since that's the fourth one for the function.  This should work: 

```
qcMWRreview(resdat,dqoaccdat,dqofrecomdat, output_dir = "D:/Documents/", output_file = "QCReport", rawdata = FALSE)
```

That being said, I did notice that the function would have returned an error if you tried the `fset` argument because I did not make the default values for the `res`, `acc`, and `frecom` arguments as `NULL`.  Should be fixed now. 

I also added an informative error to address your second issue.
